bundles/org.eclipse.jface.text/src/org/eclipse/jface/text/contentassist/AsyncCompletionProposalPopup.java

```diff
@@ -113,10 +113,9 @@ public String getDisplayString() {
 
 		@Override
 		public String getAdditionalProposalInfo() {
-			 return NLS.bind(JFaceTextMessages.getString("AsyncCompletionProposalPopup.computingDetails"), new Object[] { //$NON-NLS-1$;
-				Integer.valueOf(fSize),
-				Integer.valueOf(fSize - fRemaining),
-				Integer.valueOf(fRemaining) });
+			 return NLS.bind(JFaceTextMessages.getString("AsyncCompletionProposalPopup.computingDetails"), Integer.valueOf(fSize),
```

Comment:
@jjohnstn Looks like the fix might need further adjustments as NON-NLS is still gone here.

Reply:
Ok.  Fixed right this time.  The code to unwrap the array did not support having an NLS comment before the first array element.

Reply:
Thanks Jeff. Looks good now.